proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10581,39 +10581,718 @@ function bufferResize_( test )
+    var src = BufferNode.from( 'string' );
+    var got = _.bufferRetype( src, U8x );
+    var expected = new U8x( [ 115, 116, 114, 105, 110, 103 ] );
+    test.identical( got, expected );
+
+    test.case = 'BufferNode to I16x';
+    var src = BufferNode.from( 'string' );
+    var got = _.bufferRetype( src, I16x );
+    var expected = new I16x([ 29811, 26994, 26478, 0, -8512, 1161 ]);
+    test.identical( got, expected );
+
+    test.case = 'BufferNode to F32x';
+    var src = BufferNode.from( 'string' );
+    var got = _.bufferRetype( src, F32x );
+    var expected = new F32x( [ 1.83193727646451e+25 ] );
+    test.identical( got, expected );
+
+    test.case = 'BufferNode to U64x';
+    var src = BufferNode.from( 'str' );
+    var got = _.bufferRetype( src, U64x );
+    var expected = new U64x( [ 2340630459507n, 8899172278733n, 2340624598336n ] );
+    test.identical( got, expected );
+
+  }
+
+  /* BufferRaw */
+
+  test.case = 'empty BufferRaw to U8x';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferRetype( src, U8x );
+  var expected = new U8x( [ ] );
   test.identical( got, expected );
 
-  test.case = 'converts and returns the new type of I8x';
-  var view1 = new I16x( [ 513, 1027, 1541 ] );
-  var got = _.bufferRetype(view1, I8x);
-  var expected = got; // [ 1, 2, 3, 4, 5, 6 ];
+  test.case = 'empty BufferRaw to I16x';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferRetype( src, I16x );
+  var expected = new I16x( [ ] );
   test.identical( got, expected );
 
-  /* - */
+  test.case = 'empty BufferRaw to F32x';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferRetype( src, F32x );
+  var expected = new F32x( [ ] );
+  test.identical( got, expected );
+
+  test.case = 'empty BufferRaw to U64x';
+  var src = new BufferRaw( 8 );
+  var got = _.bufferRetype( src, U64x );
+  var expected = new U64x( [ ] );
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'BufferRaw to U8x';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferRetype( src, U8x );
+  var expected = new U8x( [ ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferRaw to I16x';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferRetype( src, I16x );
+  var expected = new I16x( [ ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferRaw to F32x';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferRetype( src, F32x );
+  var expected = new F32x( [ 1, 2, 3 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferRaw to U64x';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferRetype( src );
+  var expected = new U64x( [ 1, 2, 3 ] );
+  test.identical( got, expected );
+
+  test.case = 'BufferRawShared to U8x';
+  var src = new BufferRawShared( 5 );
+  var got = _.bufferRetype( src,  U8x );
+  var expected = new U8x( [ 0, 0, 0, 0, 0, ] );
+  test.identical( got, expected );
+
+
+  /* BufferTyped */
+
+  /* I8x */
+
+  test.case = 'empty I8x to U8x';
```

Comment:
This group has 8 test cases. The src-s have little difference. One of them is filled, the other is empty ( only initialed ).

Please, change each group. Each group should have 4 test cases. Two of them is filled, the other - empty.